eras/shelley-ma/impl/src/Cardano/Ledger/ShelleyMA/Timelocks.hs

```diff
@@ -172,12 +178,15 @@ newtype Timelock era = TimelockConstr (MemoBytes TimelockRaw era)
   deriving (Eq, Generic)
   deriving newtype (ToCBOR, NoThunks, NFData, SafeToHash)
 
+instance Memoized Timelock where
+  type RawType Timelock = TimelockRaw
+
 deriving instance HashAlgorithm (HASH (EraCrypto era)) => Show (Timelock era)
 
 type instance SomeScript 'PhaseOne (ShelleyMAEra ma c) = Timelock (ShelleyMAEra ma c)
 
 -- | Since Timelock scripts are a strictly backwards compatible extension of
--- Multisig scripts, we can use the same 'scriptPrefixTag' tag here as we did
+-- MultiSig scripts, we can use the same 'scriptPrefixTag' tag here as we did
```

Comment:
Multisignature is actually one word, afaik :bicyclist: :house:

Reply:
I didn't even think about it. For what its worth it can be with the hyphen:

> Multisignature (also multi-signature or multisig)

Regardless of the actual word name I was going by the type name:

https://github.com/input-output-hk/cardano-ledger/blob/94d1a85919edeff0a2c38a551fe7ed4c922f71eb/eras/shelley/impl/src/Cardano/Ledger/Shelley/Scripts.hs#L87